index.html

```diff
@@ -33,17 +33,17 @@ <h1>I'm Kenneth Glad to see you!</h1>
           </p>
           <a id="connect" href="index.html">LET'S CONNECT</a>
           <ul class="social-media-links">
-            <li><a href=""> <img src="images/twitter.png" alt="Twitter"/></a></li>
-            <li><a href=""> <img src="images/Linkedin.png" alt="LinkedIn"/></a></li>
-            <li><a href=""> <img src="images/medium.png" alt="Media"/></a></li>
-            <li><a href=""> <img src="images/Vector-1.png" alt="image"/></a></li>
-            <li><a href=""> <img src="images/Vector-3.png" alt="image"/></a></li>
+            <li><a href=""> <img src="images/Twitter.svg" alt="Twitter"/></a></li>
+            <li><a href=""> <img src="images/LinkedIn.svg" alt="LinkedIn"/></a></li>
+            <li><a href=""> <img src="images/Medium.svg" alt="Media"/></a></li>
+            <li><a href=""> <img src="images/Github.svg" alt="image"/></a></li>
+            <li><a href=""> <img src="images/Angellist.svg" alt="image"/></a></li>
```

Comment:
Kindly update the `alt` text according to the guidelines [here](https://www.w3.org/WAI/test-evaluate/preliminary/#images)

![image](https://user-images.githubusercontent.com/73714615/188672109-6ab91682-634c-46fb-949e-cac11df053bd.png)